Applications/Piano/Music.h

```diff
@@ -16,6 +16,11 @@ enum PianoKey {
     K_C2, K_Db2, K_D2, K_Eb2, K_E2, K_F2, K_Gb2, K_G2,
 };
 
+const enum PianoKey piano_keys[] = {
```

Comment:
Why do you need "enum" here? I don't think I've ever seen this.

Reply:
This whole construct might be unnecessary - the goal was to produce something I could "count" later on with `sizeof` hacks to allocate `m_key_rects`. This is probably not the most obvious way to do that, so I think I'll remove this whole section and use a vector instead.

Reply:
Yep, this works totally differently now.